pkg/rtc/participant.go

```diff
@@ -1824,7 +1824,10 @@ func (p *ParticipantImpl) mediaTrackReceived(track *webrtc.TrackRemote, rtpRecei
 		}
 
 		ti.MimeType = track.Codec().MimeType
-		ti.Version = p.params.VersionGenerator.New().ToProto()
+		if ti.Version == nil || utils.NewTimedVersionFromProto(ti.Version).IsZero() {
```

Comment:
no need to check version here `utils.NewTimedVersionFromProto(nil).IsZero()` is true